packages/camp/src/Input/Input.tsx

```diff
@@ -30,13 +29,18 @@ export const Input = forwardRef<InputProps, 'input'>((props, ref) => {
   // Omit extra props so they will not be passed into the DOM and trigger
   // React warnings.
   const inputProps = omit(props, ['isSuccess', 'isPrefilled'])
+  const dataAttributes = {
+    'data-is-success': props.isSuccess || undefined,
```

Comment:
we should call this `data-success` and `data-prefilled` instead. the presence of a `data-*` prop  already signifies truthiness (actually setting data-*=false is still `true`).

we might want to use `dataAttr` function to set those attributes too.

Reply:
[60201d4](https://github.com/opengovsg/camp-design-system/pull/861/commits/60201d4ebf254cb8e597a338291c80457e36377d) and [b742ce9](https://github.com/opengovsg/camp-design-system/pull/861/commits/b742ce9e273aeab5ed13dc172c06f0da7abdcdc7)